src/command-line/add.js

```diff
@@ -1,14 +1,21 @@
 "use strict";
 
-var ClientManager = new require("../clientManager");
-var colors = require("colors/safe");
-var program = require("commander");
-var Helper = require("../helper");
+const colors = require("colors/safe");
+const program = require("commander");
+const fs = require("fs");
+const Helper = require("../helper");
 
 program
 	.command("add <name>")
 	.description("Add a new user")
 	.action(function(name) {
+		if (!fs.existsSync(Helper.USERS_PATH)) {
+			log.error(`${Helper.USERS_PATH} does not exist.`);
+			return;
+		}
+
+		const ClientManager = require("../clientManager");
```

Comment:
Because running any CLI command requires all the subcommands, which in turn would require `ClientManager`, and it would cascade all the way down into loading all the server code before we would want to do it (e.g. before starting the server). This was one of the fixes for not creating config file incorrectly.